features/networking/pod.feature

```diff
@@ -151,15 +155,15 @@ Feature: Pod related networking scenarios
       | name=iperf-pods |
     And evaluation of `pod.name` is stored in the :iperf_client clipboard
     And evaluation of `pod.node_name` is stored in the :iperf_client_node clipboard
+    And evaluation of `pod.node_name(user: user)` is stored in the :node_name clipboard
```

Comment:
`(user: user)` shouldn't be needed. it is the default anyway.

Reply:
Thanks Akostadinov, removed (user: user).